### PR TITLE
Add canonical host.

### PR DESCRIPTION
**Pull request by ArturMinnullin**
Related to https://github.com/fs/rails-base/issues/298

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Change Log
 
 ## Unreleased
+- Add `gem 'rack-canonical-host'` for the definition a single host name as the canonical host for your application.
```
Comment (timurvafin) [inline CHANGELOG.md]: Please provide link to the gem `[rack-canonical-host](http://github.com/...)`

**Comment by timurvafin** [comment]
Please mention env variable in the `.env.example` with descriptive comment above.

**Comment by timurvafin** [comment]
:ok: to me, @vast ?

**Comment by timurvafin** [comment]
@ArturMinnullin https://github.com/fs/rails-base/pull/304 was merged in to `master` recently. Could you please format `.env.example` using new style?

After that please cleanup commits and we will be ready to merge.

Thanks

**Comment by ArturMinnullin** [comment]
@timurvafin done. Ready for another look.

**Pull request merged**